lua/neotest/lib/treesitter/init.lua

```diff
@@ -92,7 +94,7 @@ end
 
 ---@class neotest.lib.treesitter.ParseOptions : neotest.lib.positions.ParseOptions
 ---@field fast? boolean Use faster parsing (Should be unchanged unless injections are needed)
----@field build_position? fun(file_path: string, source: string, captured_nodes: table<string, userdata>): neotest.Position|neotest.Position[]|nil Builds one or more positions from the captured nodes from a query match.
+---@field build_position? fun(file_path: string, source: string, captured_nodes: table<string, userdata>, metadata: vim.treesitter.query.TSMetadata): neotest.Position|neotest.Position[]|nil Builds one or more positions from the captured nodes from a query match.
```

Comment:
Would love feedback on naming here! I think `metadata` may be too vague? Maybe it should be `node_metadata` or `nodes_metadata`?

Reply:
since its the name of the type, possibly treesitter_metadata?